src/py/covidr/app.py

```diff
@@ -1,10 +1,12 @@
 import os
 
 from covidr.config import DevelopmentConfig
-from flask import Flask
+from flask import Flask, send_from_directory
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Would prefer using pathlib over os.path. Also, is the realpath part necessary?

Reply:
Change to using Path. And a relative path does work, I was using absolute paths when debugging and got into the habit.